core/invoker/src/main/scala/whisk/core/container/ContainerPool.scala

```diff
@@ -363,7 +363,7 @@ class ContainerPool(
             toBeDeleted
         }
 
-        toBeDeleted.foreach(toBeRemoved.offer(_))
+        toBeDeleted.foreach { (ci : ContainerInfo) => toBeRemoved.offer(RemoveJob(false, ci)) }
```

Comment:
That's weird.  I thought I had removed them.   I added them because I was surprised that the compiler could not infer the type when just "_" was used.